pepper-apis/dsm-core/src/main/resources/liquibase/DDP-1205_CleanupDeletedOncHistoryEtc.sql

```diff
@@ -0,0 +1,116 @@
+     from ddp_onc_history_detail where deleted = 1);
+
+-- insert deleted tissues
+insert into deleted_object (original_table, original_primary_key, data)
+    (select 'ddp_tissue',
+            tissue_id,
+            json_object('tissue_id',tissue_id,'onc_history_detail_id',onc_history_detail_id,'notes',notes,'count_received',count_received,'h_e_count',h_e_count,'scrolls_count',scrolls_count,'blocks_count',blocks_count,'uss_count',uss_count,'tissue_type',tissue_type,'tissue_site',tissue_site,'tumor_type',tumor_type,'h_e',h_e,'pathology_report',pathology_report,'collaborator_sample_id',collaborator_sample_id,'block_sent',block_sent,'block_id_shl',block_id_shl,'shl_work_number',shl_work_number,'scrolls_received',scrolls_received,'sk_id',sk_id,'sm_id',sm_id,'first_sm_id',first_sm_id,'sent_gp',sent_gp,'tissue_sequence',tissue_sequence,'tumor_percentage',tumor_percentage,'additional_tissue_value',additional_tissue_value,'additional_tissue_value_json',additional_tissue_value_json,'last_changed',last_changed,'changed_by',changed_by,'deleted',deleted,'expected_return',expected_return,'return_fedex_id',return_fedex_id,'return_date',return_date)
+     from ddp_tissue where deleted = 1);
+
+-- insert tissues that probably should have been deleted by virtue of their reference to deleted onc histories.
+-- hypothesis is that these are tissues that should have been deleted when the onc history was
+-- deleted, but a bug prevented proper deletion
+insert into deleted_object (original_table, original_primary_key, data)
+    (select 'ddp_tissue',
+            tissue_id,
+            json_object('tissue_id',tissue_id,'onc_history_detail_id',onc_history_detail_id,'notes',notes,'count_received',count_received,'h_e_count',h_e_count,'scrolls_count',scrolls_count,'blocks_count',blocks_count,'uss_count',uss_count,'tissue_type',tissue_type,'tissue_site',tissue_site,'tumor_type',tumor_type,'h_e',h_e,'pathology_report',pathology_report,'collaborator_sample_id',collaborator_sample_id,'block_sent',block_sent,'block_id_shl',block_id_shl,'shl_work_number',shl_work_number,'scrolls_received',scrolls_received,'sk_id',sk_id,'sm_id',sm_id,'first_sm_id',first_sm_id,'sent_gp',sent_gp,'tissue_sequence',tissue_sequence,'tumor_percentage',tumor_percentage,'additional_tissue_value',additional_tissue_value,'additional_tissue_value_json',additional_tissue_value_json,'last_changed',last_changed,'changed_by',changed_by,'deleted',deleted,'expected_return',expected_return,'return_fedex_id',return_fedex_id,'return_date',return_date)
```

Comment:
I generated these params to `json_object()` by running one of the commented out queries at the bottom of the file.  So many columns...